src/index.ts

```diff
@@ -121,8 +121,8 @@ app.use(async (ctx: Context, next: () => Promise<unknown>) => {
 
   const sanitizedHeaders = sanitizeHeaders(resp.headers);
   if (request.method === "GET") {
-    const contentTypeHeader = sanitizedHeaders.get('Content-Type') || sanitizeHeaders.get('content-type');
-    if (contentTypeHeader.includes('text/html')) {
+    const contentTypeHeader = sanitizedHeaders.get('Content-Type');
```

Comment:
Is it safe to ignore the lower case variant? Previous code was handling case sensitivity

Reply:
`Headers.get()` should be case-insensitive: https://developer.mozilla.org/en-US/docs/Web/API/Headers/get#syntax